src/api/User/types.ts

```diff
@@ -7,11 +7,21 @@
 import {GenericObject, IDataSourceParams, PersistedItem} from '../commonTypes';
 import {AccessRight, Action, AnyAction} from '../AccessRight';
 import {IGuestPreferencesConfig} from '../Config';
+import {CaseStatus, FullCaseListSortBy} from '../Alert';
+
+export interface IUnifiedCaseListPreferences {
+  alertIdsFilter?: number[];
+  caseStatusesFilter: CaseStatus[];
+  assignedUserIdsFilter?: number[];
+  sortBy: FullCaseListSortBy[];
+  alertFoldersFilter?: number[];
```

Comment:
could we call it folderIdsFilter (like alertIdsFilter, etc)

Reply:
@EricMallard 
how about `alertFolderIdsFilter` ?
 just to distinguish between alert folders and visualisation folders